dev/integration_tests/deferred_components_test/run_release_test.sh

```diff
@@ -32,14 +32,15 @@ java -jar $bundletool_jar_path install-apks --apks=build/app/outputs/bundle/rele
 
 $adb_path shell "
 am start -n io.flutter.integration.deferred_components_test/.MainActivity
-sleep 20
+sleep 30
 exit
 "
-$adb_path logcat -d -t "$script_start_time" -s "flutter" > build/app/outputs/bundle/release/run_logcat.log
+$adb_path logcat -d -t "$script_start_time" > build/app/outputs/bundle/release/run_logcat.log
```

Comment:
is there a benefit to writing the logs to a file?

Reply:
I use the results both in the script as well as logged out for debug. This just guarantees we are working with the same set of logs. How much text can we pipe into a variable? These sets of logs can be quite large.